modules/test/playwright/tests/configuration-admin-web/instanceSettings.spec.ts

```diff
@@ -88,8 +99,77 @@ test('LPD-35562 Enter reserved screen name', async ({page}) => {
 
 	await page.getByLabel('Last Name').fill(lastName);
 
-	await instanceSettingsPage.saveAndWaitForAlert(
-		'Error:The screen name you requested is reserved.',
-		{autoClose: false, type: 'danger'}
-	);
+	await instanceSettingsPage.saveAndWaitForAlert({
+		autoClose: false,
+		text: 'Error:The screen name you requested is reserved.',
+		type: 'danger',
+	});
+});
+
+test('LPD-38043 Assert that a site configuration overrides its instance version', async ({
+	accessibilityMenuPage,
+	instanceSettingsPage,
+	page,
+	site,
+	siteSettingsPage,
+}) => {
+	await test.step('Make sure the instance accessibility configuration is disabled', async () => {
+		await instanceSettingsPage.goToInstanceSetting(
+			'Accessibility',
+			'Accessibility Menu'
+		);
+
+		if (
+			await accessibilityMenuPage.enableAccessibilityMenuCheckbox.isChecked()
+		) {
+			await accessibilityMenuPage.enableAccessibilityMenuCheckbox.uncheck();
+
+			await instanceSettingsPage.saveAndWaitForAlert({});
+		}
+	});
+
+	await test.step('Make sure the accessibility menu is not accessible in the site scope', async () => {
+		await siteSettingsPage.goToSiteSetting(
+			'Accessibility',
+			'Accessibility Menu',
+			site.friendlyUrlPath
+		);
+
+		await page.waitForLoadState();
+
+		await expect(
+			accessibilityMenuPage.openAccessibilityMenuButton
+		).not.toBeAttached();
+	});
+
+	await test.step('Enable the site accessibility configuration', async () => {
+		await expect(async () => {
+			await accessibilityMenuPage.enableAccessibilityMenuCheckbox.check();
+
+			await expect(
+				accessibilityMenuPage.enableAccessibilityMenuCheckbox
+			).toBeChecked();
+		}).toPass();
```

Comment:
This seems like a situation where we shouldn't be doing the expect().toPass() loop since it could hide issues. 

Also can you use accessibilityMenuPage.enableAccessibilityMenu() and then do the assertion after that?

Reply:
There is a hydration problem I noticed that makes the checkbox not fire the event needed to send "true" in the request. 

I added a 1s timeout and I am using `accessibilityMenuPage.enableAccessibilityMenu()`